engine.js

```diff
@@ -92,7 +92,7 @@ module.exports = function(options) {
           name: 'subject',
           message: function(answers) {
             return (
-              'Write a short, imperative tense description of the change (max ' +
+              'Write a short description of the change as a command/instruction (max ' +
```

Comment:
@jimthedev How's this instead?  I think this is better than "Write a short, imperatively moody description..." ;P

https://chris.beams.io/posts/git-commit/#imperative